commit 6dcd7540d5171e71260f54150ac6b5e43cf75acc (refs/changes/56/1556/3)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2017-02-27T21:44:36+00:00 (2 years, 7 months ago)
    
    NAPI-394 update ProvisionNic docs for 'cn_uuid' to mention being required for fabric networks (fixed typo)
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Dave Eddy <dave.eddy@joyent.com>

diff --git a/docs/index.md b/docs/index.md
index f9220d9..e19bdef 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -456,7 +456,7 @@ Creates a new NIC, provisioning an IP and MAC address in the process. The
 | Field             | Type                   | Description                                                                      |
 | ----------------- | ---------------------- | -------------------------------------------------------------------------------- |
 | owner_uuid        | UUID                   | Nic Owner                                                                        |
-| cn_uuid  |        | UUID                   | The UUID of the CN this NIC is on (optional, unless on fabric)                  |
+| cn_uuid           | UUID                   | The UUID of the CN this NIC is on (optional, unless on fabric)                  |
 | belongs_to_uuid   | UUID                   | The UUID of what this Nic belongs to                                             |
 | belongs_to_type   | String                 | The type that this belongs to (eg: 'zone', 'server')                             |
 | ip                | String                 | IP address to assign to the nic                                                  |
